src/pages/rest/b2b/negotiable-by-seller-workflow.md

```diff
@@ -0,0 +1,501 @@
+
+`POST <host>/rest/default/V1/negotiableQuote/draft`
+
+**Headers:**
+
+```terminal
+Content-Type application/json
+Authorization Bearer <admin token>
+```
+
+<CodeBlock slots="heading, code" repeat="2" languages="JSON, JSON" />
+
+#### Payload
+
+```json
+{
+  "customerId": 5
+}
+```
+
+#### Response
+
+```json
+`31` // An integer indicating the new quoteId
+```
+
+### Update negotiable quote
+
+Seller can now add products and optionally change negotiable quote name
+
+**Headers:**
+
+```terminal
+Content-Type application/json
+Authorization Bearer <admin token>
+```
+
+**Endpoint:**
+
+`PUT /V1/negotiableQuote/31`
+
+<CodeBlock slots="heading, code" repeat="2" languages="JSON, JSON" />
+
+#### Payload
+
+```json
+{
+  "quote": {
+    "id": 31,
+    "items": [
+      {
+        "sku": "24-UG02",
+        "qty": 1
+      },
+      {
+        "sku": "24-MB01",
+        "qty": 100
+      }
+    ],
+    "extension_attributes": {
+      "negotiable_quote": {
+        "quote_name": "5% for bags"
+      }
+    }
+  }
+}
+```
+
+#### Response
+
+```json
+// an empty array
+```
+
+### Adjust the negotiable quote
+
+The seller provides a 5% discount. The `negotiated_price_type` value of `1` indicates a percentage discount.
+
+**Headers:**
+
+```terminal
+Content-Type application/json
+Authorization Bearer <admin token>
+```
+
+**Endpoint:**
+
+`PUT /V1/negotiableQuote/31`
+
+<CodeBlock slots="heading, code" repeat="2" languages="JSON, JSON" />
+
+#### Payload
+
+```json
+{
+  "quote": {
+      "id": 31,
+      "extension_attributes": {
+        "negotiable_quote": {
+         "negotiated_price_type": 1,
+          "negotiated_price_value": 5
+        }
+      }
+    }
+}
+```
+
+#### Response
+
+```json
+// an empty array
+```
+
+### Recalculate prices
+
+Prices must be recalculated before sending quote to buyer.
+
+**Headers:**
+
+```terminal
+Content-Type application/json
+Authorization Bearer <admin token>
+```
+
+**Endpoint:**
+
+`POST /V1/negotiableQuote/pricesUpdated`
+
+<CodeBlock slots="heading, code" repeat="2" languages="JSON, JSON" />
+
+#### Payload
+
+```json
+{
+  "quoteIds": [31]
+}
+```
+
+#### Response
+
+```json
+`true` // indicating the request was successful
+```
+
+### Send the negotiable quote to the buyer
+
+Now that the seller has updated the quote, it must be sent to the buyer. The buyer will then be able to either accept the offer and begin the checkout process, or request further negotiations.
+
+This call places the quote in the `submitted_by_admin` state.
+
+**Headers:**
+
+```terminal
+Content-Type application/json
+Authorization Bearer <admin token>
+```
+
+**Endpoint:**
+
+`POST /V1/negotiableQuote/submitToCustomer`
+
+<CodeBlock slots="heading, code" repeat="2" languages="JSON, JSON" />
+
+#### Payload
+
+```json
+{
+  "quoteId": 31,
+  "comment": "We have provided a 5% discount to your order."
+}
+```
+
+#### Response
+
+```json
+`true` // indicating the request was successful
+```
+## Complete a negotiable quote
```

Comment:
So in the overview above, the flow is just for the Seller workflow to create the draft quote, update it, and send to the buyer.

Do you need to include the buyer flow to complete the order here? Isn't that covered in another topic that you can link to?

Also, the other flow would be for buyer to send the quote back to the Seller -- is that covered in another topic?   If not, then you could add the return for further negotiation scenario here, and link to the other flow (accept terms and complete order)

Reply:
> Do you need to include the buyer flow to complete the order here? Isn't that covered in another topic that you can link to?

There are steps to complete the order by buyer. The API calls may look similar but difference is in credentials used for call:
```
Content-Type application/json
Authorization Bearer <customer token>
```

> Also, the other flow would be for buyer to send the quote back to the Seller -- is that covered in another topic? 

I don't see it is covered anywhere. Moreover I don't see that it is possible to send Quote back to Seller using REST API. There is no endpoint that will allow to do that.